.github/workflows/test-kctrl-gh.yml

```diff
@@ -0,0 +1,63 @@
+name: test-kctrl-gh
+"on":
+  push:
+    branches:
+      - develop
+    paths-ignore:
+    - 'docs/**'
+    - '*.md'
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths-ignore:
+    - 'docs/**'
+    - '*.md'
+
+jobs:
+  test-all:
+    name: Test GH
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v1
+      with:
+        go-version: "1.17"
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+      with:
+        path: src/github.com/${{ github.repository }}
+    - name: Install Carvel Tools
+      uses: vmware-tanzu/carvel-setup-action@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        only: ytt, kapp
+        kapp: v0.43.0
```

Comment:
I wonder if these versions are bumped periodically
cc: @DennisDenuto

Reply:
> I wonder if these versions are bumped periodically

They are.

Reply:
Praveen verified that that's what we do 💯 :

Reply:
> I wonder if these versions are bumped periodically

They are.